ansible/roles/status-report-install/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Create application folder
+  file:
+    name: "/opt/status/src"
+    state: directory
+    owner: "{{ ansible_user }}"
+- name: Retrieve application source files
+  git:
+    clone: yes
+    force: yes
+    dest: "/tmp/status"
+    repo: "http://github.com/jimrigsbee/api.git"
+- name: Copy application files
+  copy:
+    dest: "/opt/status/src/{{ item }}"
+    src: "/tmp/status/{{ item }}"
+    owner: "{{ ansible_user }}"
+    mode: 0644
+    remote_src: true
+  with_items:
+    - requirements.txt
+    - server.py
+    - wsgi.py
+- name: Upgrade pip
+  pip:
+    name: pip
+    state: latest
+- name: Install python virtualenv
+  pip:
+    name: virtualenv
+    state: latest
+- name: Allow nginx to access non-standard port
+  shell: |
+    semanage port -a -t http_port_t -p tcp 30904
+    semanage port -m -t http_port_t -p tcp 30904
+
+- name: Install python application as a service
+  include_role:
+    name: Stouts.wsgi
+  vars:
+    python_enabled: false
+    wsgi_group: nginx
+    wsgi_nginx_servernames: "{{ bastion_dns_name }}"
+    wsgi_nginx_port: 30904
+    wsgi_applications:
+    - name: status
+      server: gunicorn
+      module: wsgi
+      pip_requirements: requirements.txt
+- name: Fixups for python virtual env created by Stouts
```

Comment:
Might be addressed in bastion/tasks changes upstream. See changes in status reporting PR.

Reply:
This requirements.txt comes from the src that is downloaded in a previous task.  It is not requirements.yml.

Reply:
We need to examine how upstream AgnosticD is dealing with python 3 - Stouts supports this we just have to set a variable here.

Reply:
**TODO**: Review in Stouts configuration the environment variable for setting the version of Python.

Reply:
I removed the fixups task in a test and it appears to work successfully. The report api status is updated successfully. I can also curl it and change the status.

Python is still v2.7.5